_pages/governance.md

```diff
@@ -461,7 +461,8 @@ This document draws heavily from the [Modern Paradigm for
 Standards](https://open-stand.org/about-us/principles/){:target="_blank"} and from other
 open-source governance documents including:
 
-- [https://docs.scipy.org/doc/numpy-dev/dev/governance/governance.html](https://docs.scipy.org/doc/numpy-dev/dev/governance/governance.html){:target="_blank"}
+- [https://numpy.org/doc/stable/dev/governance/index.html](https://numpy.org/doc/stable/dev/governance/index.html){:target="_blank"}
+- [https://docs.scipy.org/doc/scipy/reference/dev/governance/governance.html](https://docs.scipy.org/doc/scipy/reference/dev/governance/governance.html){:target="_blank"}
```

Comment:
here I wasn't sure whether we were referring to numpy or scipy, but given that both have been used to inform our governance, I added both as links.

the previous link was broken